ext/reflection/php_reflection.stub.php

```diff
@@ -577,7 +621,7 @@ public function isVariadic(): bool {}
 
     public function isPromoted(): bool {}
 
-    /** @tentative-return-type */
+    /** @return array<int, ReflectionAttribute> */
```

Comment:
I've just noticed that I inadvertently made the return type of all the `getAttributes()` methods tentative :/